tests.py

```diff
@@ -93,7 +93,7 @@ def test_replace_urls(self):
     def test_replace_urls_inplace(self):
         number_moved_list=use_files_dir_inplace.replace_urls(verbose=1, inplace=True)
         self.assertEqual(number_moved_list[0], 1)
-        self.assertEqual(number_moved_list[1], 1)
+        self.assertEqual(number_moved_list[1], 0)
```

Comment:
Hi, 

Thanks for this. Wondering if it is possible to test that this works before the replacement. Current tests will test only if the file has already changed.

Reply:
Hello @Nelson-Gon ,
Is this regarding testing the temp file?

Reply:
Hi, I took a second look yesterday and it seems the first test does what is expected. I just thought testing against 0 might be more problematic but since we're doing inplace replacements, I guess it's fine.  Could you please resolve conflicts since I pushed some more commits?

Thanks @spradhan92

Reply:
Will do! 

Thanks @Nelson-Gon